### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by khrystyna-sabat**
[DEMO LINK](https://khrystyna-sabat.github.io/react_tabs/)

### CODE REVIEW

**Review by k0nstrukt-or — APPROVED** [review]
Well done!

File: src/App.tsx [hunk 1]
```diff
@@ -8,33 +10,25 @@ export const tabs = [
...
+      setSelectedTab(tab);
+    }
+  };
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is&nbsp;Tab 1
+        {`Selected tab is
+          ${selectedTab.title}`}
```
Comment (illiarerun) [inline src/App.tsx:25]: you can write in one line

File: src/components/Tabs/Tabs.tsx [hunk 2]
```diff
@@ -1 +1,38 @@
...
+            key={tab.id}
+            className={classNames({ 'is-active': tab.id === selectedTab.id })}
+          >
+            <a
+              href={`#${tab.id}`}
+              onClick={() => (
+                selectedTab.id !== tab.id && (
+                  onTabSelected(tab)
+                )
+              )}
```
Comment (illiarerun) [inline src/components/Tabs/Tabs.tsx:25]: you already have this logic in parent handler
```suggestion
                onTabSelected(tab)
              )}
```

**Review by illiarerun — CHANGES_REQUESTED** [review]

File: src/App.tsx [hunk 3]
```diff
@@ -1,5 +1,7 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+import { Tabs } from './components/Tabs/Tabs';
+import { Tab } from './types/Tab';
```
Comment (mykola-koval) [inline src/App.tsx:4]: you can also store types in react-app.env.d.ts and use them without imports

**Pull request closed without merging**